desktop/core/src/desktop/settings.py

```diff
@@ -498,8 +498,14 @@
   else:
     TRUSTED_ORIGINS += desktop.conf.KNOX.KNOX_PROXYHOSTS.get()
 
+CSRF_TRUSTED_ORIGINS = []
 if TRUSTED_ORIGINS:
-  CSRF_TRUSTED_ORIGINS = TRUSTED_ORIGINS
+  for origin in TRUSTED_ORIGINS:
+    if 'http://' in origin or 'https://' in origin:
+      CSRF_TRUSTED_ORIGINS.append(origin)
+    else:
+      CSRF_TRUSTED_ORIGINS.append('http://%s' % origin)
+      CSRF_TRUSTED_ORIGINS.append('https://%s' % origin)
```

Comment:
I'm a bit confused here @amitsrivastava , so if the `origin` is present in `TRUSTED_ORIGINS, we are always appending one value in `CSRF_TRUSTED_ORIGINS` else we are always appending two values for the same origin? Any reason why?

Reply:
If the config has been provided with `http` or `https`, we will honor it and use it as is. If the config doesn't come with `http` as is the current case, we add both `http` and `https` to avoid any breaking changes and be backwards compatible.